src/moneywiz_api/model/transaction.py

```diff
@@ -362,10 +408,10 @@ def validate(self):
         assert self.original_exchange_rate is not None
 
         # assert self.amount ==  self.original_amount # original_amount could be different with amount ZCURRENCYEXCHANGERATE is playing up
+
         assert self.original_amount == pytest.approx(
-            -self.sender_amount * self.original_exchange_rate
-            - (self.original_fee or 0),
```

Comment:
The fee need to be included for sure.

Reply:
You're probably right, but Crypto shows an `AssertionError` as `sender_amount == original_amount` while I have non-zero `original_fee`. 

How do you recommend making an exception for Crypto/FX accounts?